packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -8,27 +8,40 @@ import { findProjectRootAsync, getProjectIdAsync } from '../../project/projectUt
 import { resolveWorkflowAsync } from '../../project/workflow';
 
 const EAS_UPDATE_URL = 'https://u.expo.dev';
+const DEFAULT_RUNTIME_VERSION: { policy: 'sdkVersion' } = { policy: 'sdkVersion' };
```

Comment:
```suggestion
const DEFAULT_RUNTIME_VERSION = { policy: 'sdkVersion' };
```
It's the same.

Reply:
It actually seems to cast it as a string:

<img width="446" alt="Screen Shot 2021-12-02 at 8 11 13 AM" src="https://user-images.githubusercontent.com/1220444/144428459-9fb77dfd-55e4-434f-968c-271af47b5c50.png">

While we want it to be `'sdkVersion' || 'nativeVersion'`:
<img width="1023" alt="Screen Shot 2021-12-02 at 8 13 46 AM" src="https://user-images.githubusercontent.com/1220444/144428862-ce25adbf-f542-435b-992e-35d3a90182ff.png">

Reply:
You can do `const DEFAULT_RUNTIME_VERSION = { policy: 'sdkVersion' } as const;`

Reply:
oh, nice!